app/javascript/controllers/item_selector_controller.js

```diff
@@ -63,7 +63,7 @@ export default class extends Controller {
           </span>
           <span class="vr"></span>
           <button data-action="${this.identifier}#unchecked" data-${this.identifier}-id-param="${item.id}" type="button" class="btn-close py-1" aria-label="Remove"></button>
-        </li>
+        </li> ${item.duedate ? `<div class="text-cardinal d-block align-self-center">Checked out - Due ${item.duedate}</div>` : ''}
```

Comment:
Is it ok to have this div as a child of the `ul` (and, does it flexbox ok)? Maybe it'd be better if it was inside the `li` and we move the rounded-pill styling inside the `li` too?